sdk/servicebus/service-bus/src/receivers/shared.ts

```diff
@@ -217,8 +219,13 @@ function settleMessage(
         ...options,
         associatedLinkName,
         sessionId: message.sessionId
+      })
+      .catch((err) => {
+        throw translateServiceBusError(err);
```

Comment:
`updateDispositionStatus()` internally has a try/catch block where the error is caught and translated. Any reason to do that again here?

Same question for the `settleMessage()` call a few lines below

Reply:
I'd need to look but I'm betting I just over-translated.